src/research.py

```diff
@@ -70,7 +95,7 @@ def __repr__(self):
         return '{} (x{}): {} ({}), {} ({})'.format(self.name, self.material_multiplier, self.attack, self.attack_skill_multiplier, self.defense, self.defense_skill_multiplier)
 
 class Armor:
-    def __init__(self, name, cost, material_multiplier, defense, defense_skill_multiplier):
+    def __init__(self, name, cost, material_multiplier, defense, defense_skill_multiplier, heavy=False):
```

Comment:
Good idea.

Maybe expand this into a weight stat in the future/add to weapons as well.